include/glow/Backends/BackendUtils.h

```diff
@@ -21,6 +21,43 @@
 #include "glow/IR/IR.h"
 
 namespace glow {
+namespace runtime {
+/// RuntimeSymbolInfo
```

Comment:
Nit: I'm not a fan of comments that just repeat the name of the thing being commented :-)